templates/data/prow-periodics-data.yaml

```diff
@@ -21,6 +21,23 @@ templates:
                 mountPath: /etc/config
                 readOnly: true
         jobConfigs:
+          - repoName: kyma-project/eventing-tools
+            jobs:
+              - jobConfig:
+                  name: eventing-tools-autobump
+                  cron: "30 * * * 1-5"
+                  slack_channel: kyma-prow-alerts
```

Comment:
I believe you want to change this name to something more suitable for you, or remove it completely.